### PR TITLE
Fix #643: ADD MtgjsonSet.Languages List of Strings

### LINKED ISSUE #643: Available languages for printed sets
Hello! A nice little data set to have would be to add the available printed languages for a set. What I mean by this is for example, Masters 25 (A25) was only released in English, Japanese, and Simplified Chinese. There are only translations for set names, but nothing currently about this topic.

### PR DESCRIPTION
**Pull request by ZeldaZach**
[ORI.json.txt](https://github.com/mtgjson/mtgjson/files/10279970/ORI.json.txt)

### CODE REVIEW

File: mtgjson5/providers/scryfall/monolith.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
 from singleton_decorator import singleton
 
-from .. import constants
-from ..mtgjson_config import MtgjsonConfig
-from ..providers.abstract import AbstractProvider
-from ..utils import retryable_session
+from ... import constants
+from ...mtgjson_config import MtgjsonConfig
+from ...providers.abstract import AbstractProvider
+from ...utils import retryable_session
```
Comment (kodawah) [inline mtgjson5/providers/scryfall/monolith.py:17]: 💠

**Review by kodawah — APPROVED** [review]
seems good, only improvement/change i'd make is exposing the standard language tag instead of the English one, but that can be separate i suppose

File: mtgjson5/providers/scryfall/monolith.py (hunk 1 above)
Comment (ZeldaZach) [reply mtgjson5/providers/scryfall/monolith.py:17]: I don't know what that emote means

**Comment by ZeldaZach** [comment]
Everything in MTGJSON has been through the English Tag compared to standard one, as it's not standard across providers

File: mtgjson5/providers/scryfall/monolith.py (hunk 1 above)
Comment (kodawah) [reply mtgjson5/providers/scryfall/monolith.py:17]: it's a dot, a diamond dot

**Pull request merged**